src/MICore/LaunchOptions.cs

```diff
@@ -248,7 +351,7 @@ public static ReadOnlyCollection<SourceMapEntry> CreateCollection(Xml.LaunchOpti
 
         public static ReadOnlyCollection<SourceMapEntry> CreateCollection(Dictionary<string, string> source)
         {
-            IList<SourceMapEntry> sourceMaps = source?.Select(x => new SourceMapEntry() { EditorPath = x.Value, CompileTimePath = x.Key }).ToList();
+            IList<SourceMapEntry> sourceMaps = source?.Select(x => new SourceMapEntry() { EditorPath = x.Key, CompileTimePath = x.Value}).ToList();
```

Comment:
Looks like you are never setting UseForBreakpoints in the SourceMapEntry. Breakpoints are the primary usage. You should either change the default to true or add the field to the json specification.